pending/github_desktop.xml

```diff
@@ -30,26 +32,26 @@
 -->
 <cleaner id="githubdesktop" os="windows">
   <label>GitHub Desktop</label>
-  <description>Code Management Program</description>
+  <description>Version control software</description>
   <option id="logs">
     <label>Logs</label>
-    <description>Delete Logs</description>
+    <description>Delete the logs</description>
     <action command="delete" search="walk.all" path="%AppData%\GitHubDesktop\logs\"/>
     <action command="delete" search="walk.all" path="%AppData%\GitHub Desktop\logs\"/>
-    <action command="delete" search="walk.all" path="%LocalAppData%\GitHubDesktop\*.log"/>
+    <action command="delete" search="glob" path="%LocalAppData%\GitHubDesktop\*.log"/>
     <action command="delete" search="glob" path="%LocalAppData%\GitHubDesktop\app-*\SquirrelSetup.log"/>
   </option>
   <option id="cache">
     <label>Cache</label>
-    <description>Delete Cache</description>
-    <action command="delete" search="walk.all" path="%AppData%\GitHubDesktop\*Cache\"/>
-    <action command="delete" search="walk.all" path="%AppData%\GitHub Desktop\*Cache\"/>
+    <description>Delete the cache</description>
+    <action command="delete" search="glob" path="%AppData%\GitHubDesktop\*Cache\"/>
+    <action command="delete" search="glob" path="%AppData%\GitHub Desktop\*Cache\"/>
   </option>
```

Comment:
On lines 47-48, `*Cache` refers to one or more directories that are not empty? If so, then the original `search="walk.all"` would work better because it will recursively delete the files.

Reply:
If you want me to help more, you could give an example of a full path either as a GitHub comment (here in the GitHub conversation) or an XML comment (in the file)

Reply:
I will fix that...